runtime/src/bank.rs

```diff
@@ -4185,6 +4185,12 @@ impl Bank {
         self.update_recent_blockhashes_locked(&w_blockhash_queue);
     }
 
+    // gating this under #[cfg(feature = "dev-context-only-utils")] isn't easy due to
+    // solana-program-test's usage...
```

Comment:
my mind is in firm state not to be derailed for fiddling with my pet subproject (dcou). lol

Reply:
naming and comment make the reasoning clear; all good